src/NServiceBus.Core/Unicast/Behaviors/LoadHandlersConnector.cs

```diff
@@ -18,7 +17,7 @@ public LoadHandlersConnector(MessageHandlerRegistry messageHandlerRegistry)
 
         public override void Invoke(LogicalMessageProcessingStageBehavior.Context context, Action<HandlingStageBehavior.Context> next)
         {
-            var handlerTypedToInvoke = messageHandlerRegistry.GetHandlerTypes(context.MessageType).ToList();
+            var handlerTypedToInvoke = messageHandlerRegistry.GetHandlersFor(context.MessageType).ToList();
```

Comment:
Isn't `GetHandlerTypesFor` the correct name? (since it's just getting the type and not the actual instance)